### DISCUSSION TITLE
Immune Gene Examination

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1301] [Q&A] [answered]
I created a file to examine all immune genes. Columns are as follows:
- Accession ID
- Gene name and species
- Protein name(s)
- Gene names
- Organism
- Gene length

It's around 300 lines, so a bit long to post in this discussion, but here's a link to that table in both [.csv](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/immune_genes/cbai_transcriptomev2.0/immune_gene_names.csv) and [.txt](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/immune_genes/cbai_transcriptomev2.0/immune_gene_names.txt) format (included the .csv because it automatically forms columns in the GitHub viewer, in case you don't want to download the file separately).

Basically, I'm looking for clear groups of immune genes. Anyone see anything interesting here?

P.S. This is for all immune genes in our mixed transcriptome (cbai_transcriptomev2.0). I'm in the process of creating similar tables for our crab-only and parasite-only transcriptomes, and will post them in this discussion when that's finished!

**Comment by kubu4** [comment]
I guess I'm not exactly sure what you're asking. Do you have GO/GOslims for these? I'd guess that would be the easiest way to identify immune genes (and, on that note, how'd you identify the list linked above?).

**Reply by aspencoyle** [reply]
Ooh, sorry for the lack of clarity on there. Yep, we have GO terms - in fact, this list of IDed by pulling all genes with the GO term for immune response.

Basically, the new goal of the Hematodinium project is more focused on simple identification and discovery of immune genes and existing pathways. My takeaway of the plan (which may be off) from pubathon is that the next step forward is to just manually go through the immune genes (linked above) and see if we can spot any pathways and group them accordingly. 

So I suppose I've really got two questions here:
1. Anyone see any paths they recognize?
2. How should I best go about identifying linked genes and pathways?

**Comment by aspencoyle** [comment]
Also here are the same tables produced for:
- Our crab-only transcriptome, [cbai_transcriptomev4.0](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/immune_genes/cbai_transcriptomev4.0/immune_gene_names.csv)
- Our hematodinium-only transcriptome, [hemat_transcriptomev1.6](https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/immune_genes/hemat_transcriptomev1.6/immune_gene_names.csv)

**Comment by sr320** [comment, marked as answer]
Let’s focus on crab only first

On Mon, Sep 27, 2021 at 11:35 AM Aidan Coyle ***@***.***>
wrote:

> Also here are the same tables produced for:
>
>    - Our crab-only transcriptome, cbai_transcriptomev4.0
>    <https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/immune_genes/cbai_transcriptomev4.0/immune_gene_names.csv>
>    - Our hematodinium-only transcriptome, hemat_transcriptomev1.6
>    <https://github.com/afcoyle/hemat_bairdi_transcriptome/blob/main/output/immune_genes/hemat_transcriptomev1.6/immune_gene_names.csv>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RobertsLab/resources/discussions/1301#discussioncomment-1392103>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABB4PN7CRQ2QSIBLAX42XZ3UEC2GNANCNFSM5E3ETQVQ>
> .
>
-- 

Steven B. Roberts, Associate Professor Associate Director - Graduate
Program Coordinator School of Aquatic and Fishery Sciences University of
Washington Fisheries Teaching and Research (FTR) Building - Office 232 1140
NE Boat Street - Seattle, WA 98105 robertslab.info - ***@***.*** - @sr320
vm:206.866.5141 - cell:360.362.3626 schedule a zoom call:
https://d.pr/gsgxVJ
